tailwind.config.cjs

```diff
@@ -1,7 +1,38 @@
 module.exports = {
   content: ["./src/**/*.tsx"],
   theme: {
-    extend: {},
-  },
+    extend: {
+      fontFamily: {
+        montserrat: ['Montserrat', 'sans-serif'],
+        cobeHeavy: ['CobeHeavy', 'sans-serif'],
+        cobeSemiBold: ['CobeSembiBold', 'sans-serif'],
+        lato: ['Lato', 'sans-serif'],
+      },
+      colors: {
+        offwhite: '#cbcbcb',
+        white: '#fff',
+        black: '#000',
+        green: '#6abd9e',
+        dark_green: '#33a852',
+        yellow: '#fabc05',
+        orange: '#fe6d01',
+        blue: '#17539F',
+        light_blue: '#4284f3',
+        red: '#ea4335',
+        light_grey: '#e1e1e1',
+        x_light_grey: '#dadada',
+        grey: '#666666',
```

Comment:
not objecting but assuming these are from cockpit aha?

Reply:
haha yeah, until we have designs + theme